cmds/boot/uinit/uinit_linux.go

```diff
@@ -8,20 +8,26 @@ import (
 	"fmt"
```

Comment:
why are you changing the file permissions?

Reply:
This was automatically modified by my IDE, will fix it.